packages/block-library/src/comment-template/index.php

```diff
@@ -24,10 +24,37 @@ function render_block_core_comment_template( $attributes, $content, $block ) {
 		return '';
 	}
 
-	$number = $block->context['queryPerPage'];
-
+	$per_page = $block->context['queryPerPage'];
 	// Get an array of comments for the current post.
-	$comments = get_approved_comments( $post_id, array( 'number' => $number ) );
+
+	$page = (int) get_query_var( 'cpage' );
+
+	$comment_args = array(
+		'number'  => $per_page,
+		'orderby' => 'comment_date_gmt',
+		'order'   => 'ASC',
+		'status'  => 'approve',
+		'post_id' => $post_id,
+		'offset'  => 0,
+	);
```

Comment:
Those values correspond exactly to the combined defaults of https://developer.wordpress.org/reference/functions/get_approved_comments/ and https://developer.wordpress.org/reference/classes/wp_comment_query/__construct/. This is why I didn't specify them, but perhaps it's better to be explicit about them like you did here. I don't really have the experience to say if one is better than the other or perhaps it doesn't matter 🤔

Reply:
Seems that doesn't matter. I took it from [comments_template()](https://developer.wordpress.org/reference/functions/comments_template/) function. It would be ideal that orderby, order and status would be options on the block editor or the discussion settings, maybe for that reason we have it that way in the template.

With `get_approved_comments()`, showing not approved comments would not be an option I guess.

We should be able to change it without any problem as `get_approved_comments()` also has the $args argument.